enhancements/windows-containers/health-management.md

```diff
@@ -112,6 +122,9 @@ N/A
   To reduce the risk of this, the `MaxConcurrentReconciles` option can be provided when initalizing the ConfigMap
   controller. This will enable the controller to process service ConfigMap events that occur while WMCO is handling
   its BYOH configuration responsibilities.
+* WMCO will not default to containerd as the container runtime until the 6.0.0 timeframe. Therefore, we will be using a 
+  runtime flag `dockerRuntime=false` introduced by the containerd enhancement to develop during the 5.y.z cycle.
+  This flag will dropped come WMCO 6.0.0.
```

Comment:
flag will be dropped

Reply:
Not sure I understand. It already says
`This flag will dropped come WMCO 6.0.0.`

Reply:
oh nvm i see